src/ripple/protocol/Quality.h

```diff
@@ -276,6 +276,40 @@ class Quality
         os << quality.m_value;
         return os;
     }
+
+    // return the relative distance (relative error) between two qualities. This is used for testing only.
+    // relative distance is abs(a-b)/min(a,b)
+    friend double
+    relativeDistance(Quality const& q1, Quality const& q2)
+    {
+        assert(q1.m_value > 0 && q2.m_value > 0);
+
+        if (q1.m_value == q2.m_value)  // make expected common case fast
+            return 0;
+
+        auto const minV = std::min(q1.m_value, q2.m_value);
```

Comment:
How about `auto const [minV, maxV] = std::minmax(q1.m_value, q2.m_value)`?

Reply:
Love it! Fixed in 593ef30bf `[fold] Fix type and use std::minmax`